### DISCUSSION TITLE
just a site id

### DISCUSSION THREAD

**Discussion by lesjtay** [pnp/powershell#411] [Q&A] [answered]
Registered (Global Admin), Authenticated (App-Only OAuth2), Connected to Graph:

Can't get a simple SharePoint Site Id. using a cmdlet. I must be missing something about the preciousness of an id. Tell me "there's a cmdlet for this".

Les

**Comment by LeonArmston** [comment, marked as answer]
Hi

I think the SiteId is what you are after & it can be retrieved using Get-PnPSite -includes Id

![image](https://user-images.githubusercontent.com/12968962/110786188-bc73a200-8263-11eb-95c8-5b8703e71f07.png)

You can then use this SiteId in pure graph queries etc...

**Reply by lesjtay** [reply]
Wow. How did I miss that page in the book I didn't read,

Thank you Leon!

**Comment by lesjtay** [comment]
Hi Leon,

Yes that is exactly what I was after. I mistakenly thought that cmdlet was asking me to supply what I needed.

Thank you,

Les

Sent from my iPhone

On Mar 13, 2021, at 7:56 AM, Leon Armston ***@***.***> wrote:

﻿

Hi

I think the SiteId is what you are after & it can be retrieved using Get-PnPSite -includes Id

[image]<https://user-images.githubusercontent.com/12968962/110786188-bc73a200-8263-11eb-95c8-5b8703e71f07.png>

You can then use this SiteId in pure graph queries etc...
